bin/opm

```diff
@@ -2195,6 +2198,37 @@ sub rebase_path ($$$) {
     return undef;
 }
 
+sub do_clean {
+    if (@_ == 0) {
+        err "no clean option specified.\n";
+    }
+    if ($_[0]  eq 'dist') {
+        my $dist_file = "dist.ini";
+        my $data = read_ini($dist_file);
+        my $default_sec = $data->{default};
+        my $dist_name = $default_sec->{name};
```

Comment:
`name` is a varible not a string literal, right? but I can't find where it is defined in `do_build`.